### PR TITLE
Fix various typos in doxygen and misc. source comments

### PR DESCRIPTION
**Pull request by luzpaz**
---
name: 🗎 Documentation and Styling
about: Adding or modifying **documentation** or **code style**.

---

**Description**

Fixing typos in doxygen and cleaning up varioys typos in source comments

**Changelog**

-Fixed doxygen and source comment typos

### CODE REVIEW

File: kratos/python_scripts/application_generator/classes/classCreator.py
Comment (loumalouomega) [inline kratos/python_scripts/application_generator/classes/classCreator.py:102]: I prefer Memeber...

**Review by loumalouomega — DISMISSED** [review]
Just comments, so a priori the code is not affected. Thanks

**Comment by loumalouomega** [comment]
> Just comments, so a priori the code is not affected. Thanks

AS this is your first contribution, it also requires the approval of a maintainer. Therefore, @KratosMultiphysics/technical-committee. @roigcarlo can you take a look?

**Comment by luzpaz** [comment]
soft bump

**Review by rubenzorrilla — DISMISSED** [review]
Many thanks!

**Comment by loumalouomega** [comment]
There is a conflict that must be solved

**Comment by luzpaz** [comment]
Conflict solved. Rebased

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Please move to kratos/python_scripts/testing/run_tests.py

**Comment by luzpaz** [comment]
Rebased on master.

> Please move to kratos/python_scripts/testing/run_tests.py

@loumalouomega that file doesn't exist on this branch.

Screenshot_20220815_201120

**Comment by loumalouomega** [comment]
> Rebased on master.
> 
> > Please move to kratos/python_scripts/testing/run_tests.py
> 
> @loumalouomega that file doesn't exist on this branch.
> 
> Screenshot_20220815_201120

Because you solve the conflict in a wrong manner. Please, add manually the file, with the corresponding corrections and I will allow the merge

**Comment by luzpaz** [comment]
> Because you solve the conflict in a wrong manner. Please, add manually the file, with the corresponding corrections and I will allow the merge

:facepalm: so sorry! Have no idea how that happened. That's actually never happened to me before until now. Reverted in 28e403d

**Review by loumalouomega — DISMISSED** [review]

**Comment by luzpaz** [comment]
shall i rebase?

**Comment by loumalouomega** [comment]
> shall i rebase?

Wait until CI passes and just merge

**Review by loumalouomega — DISMISSED** [review]

**Comment by luzpaz** [comment]
Let me know if rebase is needed

**Comment by roigcarlo** [comment]
Can someone re-aprove?

**Review by loumalouomega — APPROVED** [review]

**Comment by roigcarlo** [comment]
I will try to merge these as they were already approved from long ago

**Pull request merged**
